File: src/main/java/org/jukeboxmc/item/behavior/ItemCookedPorkchop.java
package org.jukeboxmc.item.behavior;

import org.jukeboxmc.item.ItemType;
import org.jukeboxmc.util.Identifier;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ItemCookedPorkchop extends ItemFood {

    public ItemCookedPorkchop( Identifier identifier ) {
        super( identifier );
    }

    public ItemCookedPorkchop( ItemType itemType ) {
        super( itemType );
    }

    @Override
    public float getSaturation() {
        return 12.8f;
    }

    @Override
    public int getHunger() {
        return 8;
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/item/behavior/ItemCookedPorkchop.kt
package org.jukeboxmc.server.item.behavior

import org.jukeboxmc.api.item.ItemType

class ItemCookedPorkchop(itemType: ItemType, countNetworkId: Boolean) : JukeboxItemFood(itemType, countNetworkId) {

    override fun getSaturation(): Float {
        return 12.8F
    }

    override fun getHunger(): Int {
        return 8
    }
}